project/Dependencies.scala

```diff
@@ -13,6 +13,7 @@ object Dependencies {
   val testInterface = "org.scala-sbt" % "test-interface" % testInterfaceVer
   val junitJupiterParams = "org.junit.jupiter" % "junit-jupiter-params" % junitJupiterVer
   val junitVintageEngine = "org.junit.vintage" % "junit-vintage-engine" % junitVintageVer
+  val junitPlatformSuite = "org.junit.platform" % "junit-platform-suite" % junitPlatformVer
```

Comment:
Alternatively, only `junit-platform-suite-api` and `junit-platform-suite-engine` are actually needed, if you prefer more specific dependencies